From: Alexis Murzeau <amubtdx@gmail.com>
Date: Thu, 24 Dec 2020 15:36:27 +0100
Subject: Remove GitHub Buttons without replacement

Upstream includes buttons to GitHub that display how
often the project was starred. Due to privacy concerns
these are removed from the documentation in Debian.

Forwarded: not-needed
Author: Stefan Breunig <stefan-debian@yrden.de>
---
 docs/_templates/sidebar/github-buttons.html | 1 -
 1 file changed, 1 deletion(-)

diff --git a/docs/_templates/sidebar/github-buttons.html b/docs/_templates/sidebar/github-buttons.html
index d0da80c..7915218 100644
--- a/docs/_templates/sidebar/github-buttons.html
+++ b/docs/_templates/sidebar/github-buttons.html
@@ -7,5 +7,4 @@
   <a href="https://github.com/streamlink/streamlink/issues"
     class="github-button"
     data-icon="octicon-issue-opened">Issues</a>
-  <script async defer id="github-bjs" src="https://buttons.github.io/buttons.js"></script>
 </div>
